Description: Documentation for systemd service files can be automatically viewed
 using systemctl help servicename if this field is present.
Author: "A. Maitland Bottoms" <bottoms@debian.org>

--- a/iiod/init/iiod.service.cmakein
+++ b/iiod/init/iiod.service.cmakein
@@ -4,6 +4,7 @@
 
 [Unit]
 Description=IIO Daemon
+Documentation=man:iiod(8)
 After=network.target
 ConditionPathExists=/sys/bus/iio
 
